src/data_sync_config.hpp

```diff
@@ -12,6 +12,8 @@
 namespace data_sync::config
 {
 
+namespace fs = std::filesystem;
```

Comment:
If there's no use of this namespace in the .hpp then I'd recommend just putting it in the .cpp so other files that include this don't get it.